CSharpSourceCode/Battle/StatusEffects/StatusEffectMissionLogic.cs

```diff
@@ -28,6 +32,93 @@ public override void OnAgentCreated(Agent agent)
             }
         }
 
+
+        public override void OnAfterMissionCreated()
+        {
+            base.OnAfterMissionCreated();
+
+            //var attacker = Mission.Current.Teams.Attacker;
+            
+            var attacker = Mission.Current.Teams.GetAlliesOf(Mission.Current.Teams.Attacker, true);
+            var defender = Mission.Current.Teams.GetAlliesOf(Mission.Current.Teams.Defender, true);
+            
+            var agents =Mission.Current.Agents;
```

Comment:
Space between = and Mission?

Reply:
did that and tidied up a bit the empty spaces